### ISSUE TITLE
Error setting up entry mbapi2020 for mbapi2020 1

### ISSUE THREAD

**Issue by fbietlot** [ReneNulschDE/mbapi2020#31]
Hello,

first of all thank you for your work.
while trying to add the integration I got the following error

Error setting up entry mbapi2020 for mbapi2020
15:49:37 – (ERROR) MercedesME 2020 (custom integration)

```
This error originated from a custom integration.

Logger: homeassistant.config_entries
Source: custom_components/mbapi2020/api.py:76
Integration: MercedesME 2020 (documentation, issues)
First occurred: 15:49:37 (1 occurrences)
Last logged: 15:49:37

Error setting up entry mbapi2020 for mbapi2020
Traceback (most recent call last):
  File "/config/custom_components/mbapi2020/api.py", line 73, in _request
    resp.raise_for_status()
  File "/usr/local/lib/python3.9/site-packages/aiohttp/client_reqrep.py", line 1004, in raise_for_status
    raise ClientResponseError(
aiohttp.client_exceptions.ClientResponseError: 404, message='Not Found', url=URL('https://bff-prod.risingstars.daimler.com/v1/vehicle/self/masterdata')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 313, in async_setup
    result = await component.async_setup_entry(hass, self)  # type: ignore
  File "/config/custom_components/mbapi2020/__init__.py", line 118, in async_setup_entry
    masterdata = await mercedes.client.api.get_user_info()
  File "/config/custom_components/mbapi2020/api.py", line 83, in get_user_info
    return await self._request("get", "/v1/vehicle/self/masterdata")
  File "/config/custom_components/mbapi2020/api.py", line 76, in _request
    raise RequestError(f"Error requesting data from {url}: {err}")
custom_components.mbapi2020.errors.RequestError: Error requesting data from https://bff-prod.risingstars.daimler.com/v1/vehicle/self/masterdata: 404, message='Not Found', url=URL('https://bff-prod.risingstars.daimler.com/v1/vehicle/self/masterdata')
```

I have already try :

- Android app is OK
- delete and readd integration
- deleted the .mercedesme-token-cache file in your HA-config folder
- try to check the "debug" option. but I don't think the integration even start... I can't see a message folder in ls config/custom_components/mbapi2020/

instalation was done via HACS 
version 
v0.5.14

running on : 
Supervised
Home Assistant 2021.12.7

Let me know if I can help and provide more info

**Comment by stitch992** [comment]
Facing same issue - opened similar issue at the same time :) . https://github.com/ReneNulschDE/mbapi2020/issues/32

**Comment by ReneNulschDE** [comment]
should be fixed with release v0.6.0
